README.md

```diff
@@ -15,14 +15,17 @@ Have fun 😃
 `/gen-build`: Generates up to 12 builds for your selected game version. *Pro Tip: You can also exclude inline-bikes if you don't like them 🙃*
 
 ## Building the bot by yourself
-If you want for whatever reason want to build the bot by yourself you can do so by pulling this repository and configure it with your bot token.
-Create a config.json inside the root folder of the project and fill it with the following.
+If, for whatever reason you want to build the bot by yourself, you can do so by pulling this repository and configuring it with your bot token. Your bot token can be found in the [Discord Developer Portal](https://discord.com/developers/applications) under your bot's name, select the "Bot" option under settings and then you should be able to see it underneath the bots username.
+
+Once you have the token, create a config.json file inside the root folder of the project and fill it with the following:
 ```JSON
 {
   "Token": "<your-bot-token-here>"
 }
 ```
 
+To invite the bot to a server, you'll need to generate a OAuth2 URL. After selecting the bot you want to add from the [Discord Developer Portal](https://discord.com/developers/applications), click on "OAuth2", then click "URL Generator". Next, you'll need to give the bot the "Bot" and "Applications.Commands" permissions under scope. Under "Bot Permissions", you'll need to select "Read Messages/View Channels", "Send Messages", and "Embed Links". At the very bottom, there will be a link that you can navigate to that will let you choose which server you want the bot to join.
```

Comment:
Please add in the first sentence to add it to a test server for testing local running builds only. There are confused people out there, that will actually confuse it with adding the love bot to their server.

Even tho that I find this section completely redundant.

People should know this already 100% as this is common sense when working or lets say developing discord bots. Anyway it can stay if you could somehow change it to what I said in my first paragraph or remove this section completely

Reply:
I'll add the "To invite the bot to a test server" part. Theres always a possibility that someone who has never worked with discord bots before could find this and want to give it a try.